soundbay/data.py

```diff
@@ -232,6 +231,46 @@ def __len__(self):
         return self.metadata.shape[0]
 
 
+
+
+class MultiClassifierDataset(ClassifierDataset):
+    '''
+    This  class inherits all the traits from ClassifierDataset and handles cases with multiple classes
+    '''
+
+    
+    def _get_audio(self, path_to_file, begin_time, end_time, label):
+        """
+        _get_audio gets a path_to_file from _grab_fields method and also begin_time and end_time
+        and returns the audio segment in a torch.tensor. it differs from ClassifierDataset _get_audio method
+        by turning the first conditional into "not background" instead of "if call"
```

Comment:
Why isn't this function suitable for the binary case?

Reply:
`(label != 0)` is a different condition then the original one `(label == 1)`
Now you mention it,
I can do it more elegant to avoid code replication